StripeFinancialConnections/StripeFinancialConnections/Source/Common/HostViewController.swift

```diff
@@ -75,7 +74,7 @@ final class HostViewController: UIViewController {
         super.viewDidLoad()
 
         view.addSubview(loadingView)
-        view.backgroundColor = .systemBackground
+        view.backgroundColor = .customBackgroundColor
```

Comment:
I was a little "forced" to change the `HostController` also because `HostController` uses `LoadingView` (where we use `LoadingView` for `WebFlowController` [which is hard-coded to one color])